client/core/bond.go

```diff
@@ -1016,18 +1019,29 @@ func (c *Core) makeAndPostBond(dc *dexConnection, acctExists bool, wallet *xcWal
 	details := fmt.Sprintf("Waiting for %d confirmations to post bond %v (%s) to %s",
 		reqConfs, bondCoinStr, unbip(bond.AssetID), dc.acct.host) // TODO: subject, detail := c.formatDetails(...)
 	c.notify(newBondPostNoteWithConfirmations(TopicBondConfirming, string(TopicBondConfirming),
-		details, db.Success, bond.AssetID, 0, dc.acct.host))
+		details, db.Success, bond.AssetID, bondCoinStr, 0, dc.acct.host))
 	// Set up the coin waiter, which watches confirmations so the user knows
 	// when to expect their account to be marked paid by the server.
 	c.monitorBondConfs(dc, bond, reqConfs)
 
 	return bond.CoinID, nil
 }
 
+func (c *Core) updatePendingBondConfs(dc *dexConnection, assetID uint32, coinID []byte, confs uint32) {
+	dc.acct.authMtx.Lock()
+	defer dc.acct.authMtx.Unlock()
+	bondIDStr := coinIDString(assetID, coinID)
+	dc.acct.pendingBondsConfs[bondIDStr] = confs
```

Comment:
Got this error here: `panic: assignment to entry in nil map`
It happened after I clicked register.

Reply:
Ah, crap. Will fix and push shortly.